src/components/Alert/__snapshots__/Alert.test.tsx.snap

```diff
@@ -77,19 +77,24 @@ exports[`Alert > renders actions 1`] = `
       style="--cpd-icon-button-size: 32px;"
       tabindex="0"
     >
-      <svg
-        class="cpd-icon"
-        fill="none"
-        height="1em"
-        viewBox="0 0 24 24"
-        width="1em"
-        xmlns="http://www.w3.org/2000/svg"
+      <div
```

Comment:
Almost all of the snapshot updates are adding a div for the extra component.

Reply:
If the `IndicatorIcon` is disabled, it's possible to not render it ? We adding an extra div in the case of the alert which is not useful

Reply:
I tried with a `<>` but I think we need something to apply the styles to in the case where its not a button.

Reply:
Oh okay